src/components/Menu/__tests__/index.tsx

```diff
@@ -4,9 +4,20 @@ import shouldMatchEmotionSnapshot from '../../../helpers/shouldMatchEmotionSnaps
 import shouldMatchEmotionSnapshotWithPortal from '../../../helpers/shouldMatchEmotionSnapshotWithPortal'
 
 describe('Menu', () => {
+  test('renders with visible=false', () =>
+    shouldMatchEmotionSnapshotWithPortal(
+      <Menu
+        visible={false}
+        baseId="menu"
+        disclosure={() => <button type="button">Menu is visible</button>}
+      >
```

Comment:
```suggestion
      <Menu
        baseId="menu"
        disclosure={() => <button type="button">Menu is visible</button>}
      >
```
`false` is its default value so no need to precise it